src/Parser/FireMage/Modules/Items/ShardOfTheExodar.js

```diff
@@ -0,0 +1,58 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const BLOODLUST_BUFFS = [
+  SPELLS.HEROISM.id,
+  SPELLS.BLOODLUST.id,
+  SPELLS.TIME_WARP.id,
+  SPELLS.NETHERWINDS.id,
+  SPELLS.ANCIENT_HYSTERIA.id,
+  SPELLS.DRUMS_OF_FURY.id,
+  SPELLS.DRUMS_OF_RAGE.id,
+  SPELLS.DRUMS_OF_THE_MOUNTAIN.id,
+];
+
+const TEAM_COOLDOWN = 600;
+const PERSONAL_COOLDOWN = 300;
+const DURATION = 40;
+
+class ShardOfTheExodar extends Module {
+
```

Comment:
Shouldn't copypasta the entire Module. We need to come up for a plan for a shared location to put this so it's the same file for Frost and Fire (and eventually Arcane)

Reply:
If we had the modules grouped by class and then spec it would have been easy enough to add a `Shared` folder for those modules. Shall I import the frost one for now?

Reply:
Leave it for now, we're discussing solutions in discord.